dirty_cat/test/test_gap_encoder.py

```diff
@@ -1,12 +1,43 @@
 import time
 import numpy as np
-from numpy.random import random
 import pytest
 import pandas as pd
 
 from sklearn.datasets import fetch_20newsgroups
 from dirty_cat import GapEncoder
 
+@pytest.mark.parametrize("init1, analyzer1, analyzer2",[
+    ('k-means++', 'char', 'word'),
+    ('random', 'char', 'word'),
+    ('k-means', 'char', 'word')
+])
+def test_analyzer(init1,analyzer1,analyzer2):
+    """" Test if the output is different when the analyzer is 'word' or 'char'.
+        If it is, no error ir raised. 
+    """
+    add_words=False
```

Comment:
PEP8: spaces on both sides of the assignment equal sign (but not the keyword param equal sign)
```suggestion
    add_words = False
```